CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+## v6.4.0 (2018-09-08):
+
+### NEW FEATURES
+
+* [`6e9f04b0b`](https://github.com/npm/cli/commit/6e9f04b0baed007169d4e0c341f097cf133debf7)
```

Comment:
wow the implementation for that ended up being waaay better than it originally was

Reply:
I agreeee